### PR TITLE
support to "import ClientMonitor from 'skywalking-client-js'" in "typescript" and "es6"

### PR DESCRIPTION
**Pull request by vihoyau**
1.Add index.d.ts to the package.json's files.
2.change the index.d.ts , use any type.
3.change the python file , exclude index.d.ts

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export as namespace ClientMonitor;
```
Comment (wu-sheng) [inline index.d.ts:1]: Why don't you add a license header here?

Comment (vihoyau) [reply index.d.ts:1]: its rabbit write，i will delete it

Comment (vihoyau) [reply index.d.ts]: sorry,i dont know how to do right, i try it

Comment (wu-sheng) [reply index.d.ts]: The tool checks you should have the Apache 2.0 header on the top of the source codes. Like other codes in the repository. You should not exclude it.

File: tools/check-license-header.py [hunk 2]
```diff
@@ -45,7 +45,7 @@
 
 comment_leading_chars = ('#', '::', '/*', '*', ' ')
 exclude_dirs = ('.git', 'lib', 'node_modules', 'release', '.idea', 'license')
-exclude_files = ('.md', '.DS_Store', 'NOTICE', 'LICENSE', '.gitignore', 'package.json', 'package-lock.json', '.prettierrc')
+exclude_files = ('.md', '.DS_Store', 'NOTICE', 'LICENSE', '.gitignore', 'package.json', 'package-lock.json', '.prettierrc', 'index.d.ts')
```
Comment (wu-sheng) [inline tools/check-license-header.py:48]: This file should not be added as exclusive.

File: index.d.ts (hunk 1 above)
Comment (vihoyau) [reply index.d.ts:20]: so,I have completed the license。

File: tools/check-license-header.py (hunk 2 above)
Comment (vihoyau) [reply tools/check-license-header.py:48]: i dont no how to solve the problem about "CI" check

Comment (vihoyau) [reply tools/check-license-header.py]: thank you ,I've solved it.

**Comment by wu-sheng** [comment]
Please correct the title of PR to match your code change.

**Comment by vihoyau** [comment]
my local "CI" is ok, why that dosen't effective

**Comment by wu-sheng** [comment]
CI shows,
1. This header seems not expected, maybe wrong comment formart for ts file? 
2. Can't pass compiling in Node 16.x.

I am not a frontend developer, but these should not be an issue.

**Comment by vihoyau** [comment]
my "node" version is v16.17.0
i use command "npm ci " 、"npm run build --if-present " 、"python3 tools/check-license-header.py ." 。
its ok
image

**Comment by wu-sheng** [comment]
You could use your own fork repo, to trigger GitHub action.
We can't know your local env.

**Comment by vihoyau** [comment]
i fork repo anew , it‘s error when take a command "npm install " .
because the webpack version. 
12.xx is ok.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -34,9 +34,9 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install and build
+    - name: yarn install and build
       run: |
-        npm ci
+        rm -rf node_modules && yarn install --frozen-lockfile
```
Comment (wu-sheng) [inline .github/workflows/build.yml:39]: I am not sure we expect this. Wait for @Fine0830

Comment (vihoyau) [reply .github/workflows/build.yml:39]: thank you , i change to wepack . can you see the last file commit?

Comment (wu-sheng) [reply .github/workflows/build.yml:39]: I am not familiar with frontend codes. Will wait for the reviewer.

Comment (vihoyau) [reply .github/workflows/build.yml:39]: can you open the issue entrance? i have some question to ask.
example:
when I use the node_module, it show the error message.
import './lib/index';
^^^^^^
SyntaxError: Cannot use import statement outside a module

Comment (wu-sheng) [reply .github/workflows/build.yml:39]: You could discuss here if it is relative to your PR.

**Review by Fine0830 — COMMENTED** [review]
I don't understand why there are so many changes here. We just need to add the `index.d.ts` and update the `files` in `package.json`.

**Pull request closed without merging**